src/components/FormSections/PersonalInfo.jsx

```diff
@@ -185,11 +228,12 @@ function PersonalInfo(props){
                             autoComplete="off"
                             required={parseInt(dobFieldState.required)}
                             data-testid="dateInputGroup"
-                            onBlur={event => { if (!event.currentTarget.contains(event.relatedTarget)) checkDateValues(); }}
-                        >
+                            onBlur={e => { if (!e.currentTarget.contains(e.relatedTarget)) toggleError(e, checkDateValues('all')) }}
```

Comment:
I am able to proceed with with a future date of birth `01 01 2025` even though an error displays.
![Screenshot 2024-02-05 at 4 16 39 PM](https://github.com/usagov/vote-gov-nvrf-app/assets/19731897/fb9ff4fe-81fa-4f15-940b-8d846f37d083)

Reply:
removed custom year validation and html5 pattern